### PR TITLE
eth.syncing update

### PR DESCRIPTION
**Pull request by verheesj**
resolves issue #23617

### CODE REVIEW

File: docs/_interface/mining.md [hunk 1]
```diff
@@ -68,7 +68,9 @@ Example output of first stage of block downloading:
       startingBlock: 3809258
 }
 
-A response from `eth.syncing` of `false` means you are synced up.
```
Comment (verheesj) [inline docs/_interface/mining.md:71]: A response `false` is only indicative of a complete sync.

File: docs/_interface/mining.md [hunk 2]
```diff
@@ -68,7 +68,9 @@ Example output of first stage of block downloading:
...
+You will import up to the `highestBlock` and `knownStates`. Block importing will stop `~64` blocks behind head and finish importing states. 
+
+Once all `states` are downloaded, `geth` will switch into a full node and sync the remaining `~64` blocks fully, as well as new ones. In this context, `eth.syncing` returns false once synced.
```
Comment (verheesj) [inline docs/_interface/mining.md:73]: Provides better understanding in regards to establishing a complete sync.

**Review by verheesj — COMMENTED** [review]
Ready

**Review by Zachinquarantine — APPROVED** [review]

**Comment by karalabe** [comment]
Thanks

**Pull request merged**